test/Emitter/SapiStreamEmitterTest.php

```diff
@@ -154,7 +154,6 @@ public function testDoesNotInjectContentLengthHeaderIfStreamSizeIsUnknown(): voi
         $stream->method('isSeekable')->willReturn(false);
         $stream->method('isReadable')->willReturn(false);
         $stream->method('eof')->willReturn(true);
-        $stream->method('rewind')->willReturn(true);
```

Comment:
This mock is not seekable so rewind must not be called. There should really be test ensuring this behavior but here it is fine to drop method mock.